hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/protocolPB/DiskBalancerProtocolPB.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hdds.protocolPB;
+
+import org.apache.hadoop.hdds.protocol.proto.DiskBalancerProtocolProtos.DiskBalancerProtocolService;
+import org.apache.hadoop.ipc.ProtocolInfo;
+
+/**
+ * Protocol that clients use to communicate directly with Datanodes
+ * for DiskBalancer operations.
+ */
+@ProtocolInfo(
+    protocolName = "org.apache.hadoop.hdds.protocol.DiskBalancerProtocol",
+    protocolVersion = 1)
+public interface DiskBalancerProtocolPB extends
```

Comment:
@KerberosInfo(serverPrincipal = HddsConfigKeys.HDDS_DATANODE_KERBEROS_PRINCIPAL_KEY)

Reply:
Define a org.apache.hadoop.hdds.protocol.DiskBalancerProtocol.acl in HddsConfigKeys and add the protocol in HddsPolicyProvider.